### PR TITLE
Update Set-CsOnlineDialInConferencingUser.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Adding a note to call out 3 parameters of this cmdlets that will be deprecated by the end of Jan.

### CODE REVIEW

**Comment by tiburd** [comment]
@get-itips @dariomws @tseward 
Hi! 
Whoever of you reviews this PR last, please ping me so that I can merge it.
Thank you!

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsOnlineDialInConferencingUser.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ ms.reviewer:
 # Set-CsOnlineDialInConferencingUser
 
 ## SYNOPSIS
+
+> [!NOTE]
+> The AllowPSTNOnlyMeetings, ResetConferenceId, and ConferenceId parameters will be deprecated on Jan 31, 2022. To allow Teams meeting participants joining via the PSTN to bypass the lobby, use the AllowPSTNUsersToBypassLobby of the [Set-CsTeamsMeetingPolicy cmdlet](https://docs.microsoft.com/powershell/module/skype/set-csteamsmeetingpolicy). The capabilities associated with the ResetConferenceId and ConferenceId parameters are no longer supported.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineDialInConferencingUser.md:18]:
```suggestion
> [!NOTE]
> The AllowPSTNOnlyMeetings, ResetConferenceId, and ConferenceId parameters will be deprecated on Jan 31, 2022. To allow Teams meeting participants joining via the PSTN to bypass the lobby, use the AllowPSTNUsersToBypassLobby of the [Set-CsTeamsMeetingPolicy cmdlet](/powershell/module/skype/set-csteamsmeetingpolicy). The capabilities associated with the ResetConferenceId and ConferenceId parameters are no longer supported.
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Enable-CsOnlineDialInConferencingUser cmdlet has the ConferenceId parameter. Is it available there?

Could we still reset the conference ID from the Teams Admin Center?

Thanks!

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by ruizoscar** [comment]
@dariomws. Eventually, yes, but not for the moment. This deprecation is phased and the cmdlet is the first phase.

**Pull request merged**